src/js/components/DataTable/Header.js

```diff
@@ -23,36 +23,37 @@ import {
   StyledDataTableHeader,
   StyledDataTableRow,
 } from './StyledDataTable';
-import { datumValue, calcPinnedBackground } from './buildState';
+import { datumValue } from './buildState';
 import { kindPartStyles } from '../../utils/styles';
 import { normalizeColor } from '../../utils/colors';
 
 // separate theme values into groupings depending on what
 // part of header cell they should style
 const separateThemeProps = theme => {
   const {
-    background,
-    border,
+    background, // covered by cellProps
+    border, // covered by cellProps
     color,
     font,
     gap, // gap is used for space between header cell elements only
+    pad, // covered by cellProps
     units,
     ...rest
   } = theme.dataTable.header;
 
-  const cellProps = { background, border };
+  // const cellProps = { background, border };
```

Comment:
Should this line just be removed instead of commented out?

Reply:
good catch, removed